ws-nextjs-app/pages/[service]/send/[id]/styles.ts

```diff
@@ -32,14 +32,27 @@ export default {
   primaryColumn: ({ mq, spacings }: Theme) =>
     css({
       gridColumn: '1 / span 12',
-
       [mq.GROUP_4_MIN_WIDTH]: {
         gridColumn: '1 / span 8',
         paddingBottom: `${spacings.QUADRUPLE}rem`,
       },
     }),
   mainContent: ({ spacings, palette, mq }: Theme) =>
     css({
+      a: {
+        fontWeight: 700,
```

Comment:
Is `fontWeight` usually something we hardcode like this? I thought this value was computed based on the `fontVariant`? e.g. https://github.com/bbc/simorgh/blob/f5233e91760bbac4c0a87d516cfe4fb9f59c6835/src/app/components/Curation/Subhead/index.styles.ts#L6 - if you provide `sansBold` as the fontVariant, then it should send the correct fontWeight based on the service and its font